### PR TITLE
Feature dockercompose integration

### PR DESCRIPTION
**Pull request by kevin-schmitt**
Hi,

I have update last pull request for avoid problem with conflict file.  Like advised by @cristophernic i change credential file with make command.

For test:
`git clone --single-branch --branch feature-dockercompose-integration https://github.com/kevin-schmitt/laconia.git`

Then run command make (i don't test for windows os :))
Edit:
> make init-containers
> make start
> make install-app

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,17 @@ Create a virtual host called `laconia.server`. The server should point to the `/
 
 Laconia is all set up and ready to use!
 
+### Or use Docker install
+- Require [docker-compose](https://docs.docker.com/compose/install/) and command  make
+- Run `make init` for init containers and config file
```
Comment (taniarascia) [inline README.md]: This command doesn't exist - your makefile has `init-containers`.

File: src/models/Database.php [hunk 2]
```diff
@@ -33,6 +33,7 @@ class Database
     public function __construct() 
     {
         $dsn = 'mysql:host=' . $this->host . ';dbname=' . $this->dbname;  
+
```
Comment (taniarascia) [inline src/models/Database.php]: Remove unnecessary white space.

File: ~/.vscode-root/Backups/workspaces.json [hunk 3]
```diff
@@ -0,0 +1 @@
+{"rootURIWorkspaces":[],"folderURIWorkspaces":["file:///home/kevin/laconia"],"emptyWorkspaceInfos":[{"backupFolder":"1565722292881"}],"emptyWorkspaces":["1565722292881"]}
```
Comment (taniarascia) [inline ~/.vscode-root/Backups/workspaces.json]: Please remove all `~/.vscode-root` files.

File: Makefile [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+start:
+	docker-compose up -d 
+
+stop:
+	docker-compose stop
+
+init-containers:
+	docker-compose up -d --build 
+	docker-compose exec $(CONTAINER_NAME) /bin/cp -f config/credentials.php config/credentials_back.php
```
Comment (taniarascia) [inline Makefile]: Does this file exist?

**Review by taniarascia — CHANGES_REQUESTED** [review]
Before this PR can be approved:

1. Please remove all .vscode configuration files
2. Make sure all instructions are correct
3. Fix all merge conflicts

Thank you!

File: .gitignore [hunk 5]
```diff
@@ -1,3 +1,6 @@
 /vendor
 /node_modules
-.DS_Store
```
Comment (kevin-schmitt) [inline .gitignore]: How i can remove it please?

**Comment by kevin-schmitt** [comment]
Thanks @taniarascia  for you review,
I have problem for fix white space (.gitignore, IndexController.php, Database.php) have you solution or idea for this please?

Thanks

**Comment by kevin-schmitt** [comment]
@taniarascia  I finish resolve conflict :)
I update readme for installation, say me if is not clear or if you find problems. 

Thanks.

**Comment by taniarascia** [comment]
I have everything working except htaccss -> nginx rewrites. Those htaccess rewrites were poorly implemented on my part, which is why it's not easy to migrate to nginx...I'll have to fix it in the future.

**Pull request merged**